src/helpers/resultSpec.ts

```diff
@@ -30,3 +31,24 @@ export function getStackingResultSpec(
     const getDimensions = isStackedChart(buckets) ? generateStackedDimensions : undefined;
     return getResultSpec(buckets, sortBy, getDimensions);
 }
+
+export function getGeneralResultSpec(afm: AFM.IAfm): AFM.IResultSpec {
```

Comment:
please move the content of this function into file `src/helpers/dimensions.ts` next to the function `generateDefaultDimensions` and consider the different name of function

Reply:
done